### PR TITLE
Minor code cleanups/fixes

### PR DESCRIPTION
**Pull request by kormat**
- Indentation fixes
- Simplified logic in write_topo_files()
- Removed unused test variables
- Consolidated 2 write+tests in write_trc_files() into one.

@rev112 
@pszalach

### CODE REVIEW

File: topology/generator.py [hunk 1]
```diff
@@ -589,9 +580,10 @@ def write_trc_files(self, ad_configs, keys):
                         core_ads, {}, registry_server_addr,
                         registry_server_cert, root_dns_server_addr,
                         root_dns_server_cert, trc_server_addr, signatures)
-                    write_file(trc_file, str(trc))
-                    # Test if parser works
-                    trc = TRC(trc_file)
+                write_file(trc_file, str(trc))
+                # Test if parser works
+                trc = TRC(trc_file)
```
Comment (tonyo) [inline topology/generator.py]: This `trc` is not used either, is it?

Comment (tonyo) [inline topology/generator.py]: Also line 600.

Comment (tonyo) [inline topology/generator.py]: Oh, and 289 too.

Comment (kormat) [inline topology/generator.py]: Good catches, fixed.

**Pull request merged**